app/assets/stylesheets/styles/components/_nav.css.scss

```diff
@@ -55,7 +61,7 @@ html, body {
       }
 
       a {
-        background: rgba(0,0,0,0.4) !important;
+        background: rgba(0,0,0,0.6) !important;
```

Comment:
!important should not be used<br>`0.6` should be written without a leading zero as `.6`<br>Commas in function arguments should be followed by a single space